uber/templates/base.html

```diff
@@ -181,6 +181,112 @@
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1">
+    {% block sectionStyle %}{% endblock %}
+    <!-- additional styles -->
+    {% block additional_styles %}{% endblock %}
+    {% if 'preregistration' not in c.PAGE_PATH %}
+        {% for js in c.JAVASCRIPT_INCLUDES %}
+            <script src="{{ js }}" type="text/javascript"></script>
+        {% endfor %}
+    {% endif %}
```

Comment:
sorry this is so messy, base.html is something I took too far in the original refactor in the original Jinja2 PR.

One thing to look out for here is (not shown in this diff) there may be an overlap for the following blocks introduced by this code in this file around line 20:
```
additional_styles
```

I regret being a little careless in the original Jinja2 PR by trying to ALSO do the following refactors, which we need but really belong in another PR because the JInja2 PR is ridiculously complex:
1) trying to cleanup the entire block structure for all the CSS/JS/HEAD stuff here in base.html
2) trying to make all pages be able to inherit from base.html and not have to have a separate base-admin.html.  The original difference between an 'admin' page and a public page was mostly turning on and off the menu, and maybe including slightly different CSS/JS.  I tried to do this with a new jinja2 feature that sets a variable named ```{% set admin_area=True %}```   you can see the differences here: https://github.com/magfest/ubersystem/pull/1888/files#diff-57600a5be4d84741f7260232ede2ab66L1

As long as these changes don't seem to break everything, I would recommend for simplicity sake go ahead and merge this PR and deal with any fallout in the next PR.   

#SOON we will get out of the bad parts of mergetown and into greener pastures.